src/Tizen.NUI.Wearable/src/public/Title.cs

```diff
@@ -328,16 +328,20 @@ private Geometry CreateQuadGeometry()
 
             TexturedQuadVertex[] texturedQuadVertexData = new TexturedQuadVertex[4] { vertex1, vertex2, vertex3, vertex4 };
 
-            int length = Marshal.SizeOf(vertex1);
-            IntPtr pA = Marshal.AllocHGlobal(checked(length * 4));
+            int size = Marshal.SizeOf(vertex1);
+            IntPtr pA = Marshal.AllocHGlobal(size * texturedQuadVertexData.Length);
```

Comment:
```suggestion
            IntPtr pA = Marshal.AllocHGlobal(checked(size * texturedQuadVertexData.Length));
```

Don't forget to use `checked` keyword, which will resolve the svace issue what we resolved before.